lib/veil/credential_collection/base.rb

```diff
@@ -146,6 +150,21 @@ def add(*args)
       end
       alias_method :<<, :add
 
+      # Add the contents of a file as a credential after
+      # verifying that the file can be read.
+      # Usage:
+      #  add_from_file(filename, "secretname")
+      #  add_from_file(filename, "groupname", "secretname")
+      #
+      #  `add`'s options are not supported
+      #
+      def add_from_file(filepath, *args)
```

Comment:
I there a way we can make this support passing `:frozen => true`? (Does it already, I may just not be seeing it).  I think that most of the time we have things coming from files right now, we want to freeze them because veil doesn't know how to rotate them yet.